### ISSUE TITLE
Search field is missing label

### ISSUE THREAD

**Issue by femmebot** [18F/nsf-sbir#696]
## Expected Behavior
- [ ] Identify form(s). Identify form elements (inputs, buttons, textbox, text area, radio, select, etc). Ensure each element has a label or some programtic name associated with it

see: 508 compliance report [Portfolio N° 26](https://docs.google.com/spreadsheets/d/1yWVeVHPJtoqsCTEwnYalqDTSXNkr5E_Bn20msVmkP-8/edit#gid=0)

**Comment by femmebot** [comment]
@line47 i think this is no longer relevant since I can see there _is_ a form label for Search? I think the review was conducted on the earlier version of search that I just pasted in?

**Comment by line47** [comment]
Yeah I'm not finding any missing labels, I'm going to close this issue.
